### ISSUE TITLE
.EMAIL TLD and DS record algorithm 8

### ISSUE THREAD

**Issue by mabkenar** [mail-in-a-box/mailinabox#1538]
First of all, thank you for making and actively maintaining such a great project. Mail-in-a-Box is not only an important and much-needed project, but it is also a delight to use. I appreciate all the hard work that you have put into Mail-in-a-Box.

My MiaB works perfectly. However, I haven't been able to get DANE working:
![image](https://user-images.githubusercontent.com/3006332/54532517-35e39d00-4988-11e9-8a7e-e440d67a4ff3.png)

I use Gandi nameservers (i.e., external DNS records). Gandi support told me

> The .EMAIL registry does not support the algorithm 8, only the algorithm RSA (5), ECDSAP256SHA256 (13) and ECDSAP384SHA384 (14) are supported. 

However, what MiaB gives me is in Status Check page is only a DS record with algorithm 8.

How can I get a DS record (to add in my registrar's control panel) which is with algorithms 5, 13, or 14?

**Comment by JoshData** [comment]
The algorithms need to be listed at https://github.com/mail-in-a-box/mailinabox/blob/master/setup/dns.sh#L91 and then the algorithm selected is TLD-dependent at https://github.com/mail-in-a-box/mailinabox/blob/master/management/dns_update.py#L529. (Ideally, we should modify the second part so that the algorithm can be chosen by the user on a domain-by-domain basis.)

There's probably no real need for supporting both ECDSAP256SHA256 and ECDSAP384SHA384, and the latter is probably overkill, so I would recommend only adding ECDSAP256SHA256 in the first source file, and then mapping .email to this algorithm in the second source file.

One missing piece of information here is how this project's website is working, since it's a .email and I'm pretty sure I have DNSSEC set up.

**Comment by just4t** [comment]
@JoshData : Looks your are DNSSEG validated in your **.email** website (verified through Firefox [DNSSEG](https://addons.mozilla.org/en-US/firefox/addon/dnssec/) add-on as follows:

![image](https://user-images.githubusercontent.com/13877636/54887477-e3c4df00-4e93-11e9-9ece-2b66037ef064.png)

Hope this helps.

**Comment by JoshData** [comment]
Yup, so, .email must support the older algorithm, but maybe I'm grandfathered-in and it's not supported for new configurations, or it's dependent on the registrar.
